src/PrestaShopBundle/Resources/config/services/core/module.yml

```diff
@@ -55,6 +55,19 @@ services:
   prestashop.module.factory.sourcehandler:
     alias: PrestaShop\PrestaShop\Core\Module\SourceHandler\SourceHandlerFactory
     public: true
-    deprecated:
-      package: PrestaShop\PrestaShop
-      version: 8.1
+    deprecated: ~
```

Comment:
We should keep the 8.1 indication to know when this was deprecated, so we can remove it in the next version
Actually, this should be removed in the v9.0 but in a dedicated PR probably